sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/AzureMonitorTransmitter.cs

```diff
@@ -36,11 +36,31 @@ public AzureMonitorTransmitter(AzureMonitorExporterOptions options, TokenCredent
 
             options.Retry.MaxRetries = 0;
 
-            _connectionVars = InitializeConnectionVars(options);
+            var connectionString = GetConnectionString(options);
+            _connectionVars = ConnectionStringParser.GetValues(connectionString);
 
             _applicationInsightsRestClient = InitializeRestClient(options, _connectionVars, credential);
 
             _fileBlobProvider = InitializeOfflineStorage(options);
+
+            // TODO: uncomment following line for enablement.
+            // InitializeStatsbeat(connectionString);
+        }
+
+        private static void InitializeStatsbeat(string? connectionString)
+        {
+            try
+            {
+                // Do not initialize statsbeat for statsbeat.
+                if (connectionString != null && connectionString != Statsbeat.Statsbeat_ConnectionString_EU && connectionString != Statsbeat.Statsbeat_ConnectionString_NonEU)
+                {
+                    _ = new Statsbeat(connectionString);
```

Comment:
Should this method return `StatsBeat?`

If nothing holds a reference to StastBeat, I would expect GC to dispose of this.

Reply:
@vishweshbankwar As we discussed offline, AzureMonitorTransmitter need to implement IDisposable and it should be disposed in exporter. `Statsbeat` object could be disposed in `AzureMonitorTransmitter` dispose.

GC will not dispose this object as it got a child object of MeterProvider instance and that won't get disposed.

Reply:
Added todo here